### PR TITLE
SED-1058 executionError.html

### PR DESCRIPTION
**Pull request by dvladir**

### CODE REVIEW

File: projects/step-frontend/src/lib/angularjs/partials/execution/executionError.html [hunk 1]
```diff
@@ -1,4 +1,15 @@
...
+  <step-execution-errors
+    [(selected-error-distribution-toggle)]="selectedErrorDistirbutionToggle"
+    [count-by-error-msg]="countByErrorMsg"
+    [count-by-error-code]="countByErrorCode"
+    [error-distribution]="errorDistribution"
+    (search-step-by-error)="searchStepByError($event)"
+  >
+  </step-execution-errors>
+
+<!--
```
Comment (neogucky) [inline projects/step-frontend/src/lib/angularjs/partials/execution/executionError.html:12]: Not a fan of commenting out code, why is this needed?

**Comment by neogucky** [comment]
Is the old code used elsewhere or why is this migration task only adding new code but not removing old one?

File: projects/step-frontend/src/lib/angularjs/partials/execution/executionError.html (hunk 1 above)
Comment (dvladir) [reply projects/step-frontend/src/lib/angularjs/partials/execution/executionError.html:12]: I left it, as it was left in other templates too.
If you think that it should be removed. Lest remove

**Review by neogucky — APPROVED** [review]
